es.json

```diff
@@ -2,194 +2,197 @@
-        "squad-revive": "squad revive",
-        "squad-unique-name": "A squad already exists with that name.",
-        "squad-player-not-in-squad": "Player [%s] is not in your squad",
-        "squad-settag-success": "Squad tag has been set to [%s]",
-        "squad-private": "Your squad is now %s.",
-        "squad-already-in-squad": "You are already in a squad, leave before joining another squad.",
-        "squad-wrong-team": "This squad is on the %s team, please switch teams to join this squad.",
-        "squad-kicked-you": "You were removed from the squad by the leader.",
-        "squad-members": "Squad members: %s",
-        "squad-unban": "Unbanned [%s] from your squad"
+        "squad-left-you": "Has abandonado tu escuadrón!",
+        "squad-cant-promote-out-of-squad": "No puedes ascender a alguien que no está en tu SQUAD!",
+        "squad-promote-to-leader": "Has sido ascendido a lider de escuadrón!",
+        "squad-invited-sender": "Has invitado a %s a tu escuadron!",
+        "squad-invited-reciever": "¡Has sido invitado al escuadrón %s! Escribe /squad accept %s para unirte.",
+        "squad-invite-own-team-error": "¡Solo puedes invitar a miembros de tu propio equipo!",
+        "squad-disbanded": "El escuadrón ha sido disuelto, has sido expulsado.!",
+        "squad-new-member": "%s se acaba de unir al escuadron (%s/%s)!",
+        "squad-already-in": "Ya estas en un escuadron!",
+        "squad-no-name": "Dale a tu escuadrón un nombre válido!",
+        "squad-created": "Se creó con éxito el escuadrón %s.",
+        "squad-no-id": "No especificaste un escuadrón para unirte (/squad accept [id])",
+        "squad-doesnt-exist": "No existe escuadrón con esa ID!",
+        "squad-no-invite": "No has sido invitado a este escuadrón!",
+        "squad-joined": "Te has unido al escuadrón %s!",
+        "squad-not-in": "No estás en un escuadrón!",
+        "squad-full": "Este escuadron esta completo",
+        "squad-no-player-invite": "No especificaste un jugador para invitar a tu escuadron!",
+        "squad-no-player-kick": "No especificaste un jugador para expulsar de su equipo!",
+        "squad-no-player-promote": "No especificaste un jugador para ascender en tu equipo!",
+        "squad-player-not-found": "El jugador especificado no se pudo encontrar o ya es parte de un equipo.",
+        "squad-kick-success": "Expulsó con éxito a %s de su escuadrón.",
+        "squad-promote-success": "Ascendió con éxito a %s a líder del escuadrón.",
+        "squad-mvp-clantag": "Necesitas tener MVP para personalizar tu etiqueta de clan!",
+        "squad-no-clantag": "No ha especificado una etiqueta para establecer!",
+        "squad-not-leader": "No eres el líder de este escuadrón.!",
+        "squad-revive": "Revivir escuadron",
+        "squad-unique-name": "Ya existe un escuadron con ese nombre.",
+        "squad-player-not-in-squad": "El jugador [% s] no está en tu escuadron",
```

Comment:
Malformed wont work on production server - please correct to [%s] then good for merge.